script.js

```diff
@@ -0,0 +1,14 @@
+const hamburger = document.querySelector(".hamburger");
+const navMenu = document.querySelector(".nav-menu");
+
+hamburger.addEventListener("click", () => {
+  hamburger.classList.toggle("active");
+  navMenu.classList.toggle("active");
+});
+
+document.querySelectorAll(".nav-link").forEach((n) =>
+  n.addEventListener("click", () => {
+    hamburger.classList.remove("active");
+    navMenu.classList.remove("active");
+  })
+);
```

Comment:
- [ ] Kindly make sure you remove the `Scroll` on the header and When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed. As for now, it is not the case on some options like (About && Contact)

- [ ] Kindly make sure your menu follows the design from [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., font, colors, images, tests, margins) as required for the project.
#### Figma design
![mf](https://user-images.githubusercontent.com/83042742/173396244-8973cdfe-585f-4d38-b3ff-84f587e81d6c.png)


#### Your design
![YD](https://user-images.githubusercontent.com/83042742/173396152-a6b96b66-5ade-473d-a3db-ae2b7b31ee03.png)